src/control/system/database_members.go

```diff
@@ -68,6 +68,23 @@ func (mam MemberAddrMap) addMember(addr *net.TCPAddr, m *Member) {
 	mam[addr.String()] = append(mam[addr.String()], m)
 }
 
+func (mam MemberAddrMap) removeMember(m *Member) {
+	members, ok := mam[m.Addr.String()]
```

Comment:
nit: ok->exists

Reply:
Good catch, I'll correct it if there are any other changes to be made.

Reply:
Fixed